components/modal/examples/00-default.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Modal, ModalHeader, ModalBody, ModalFooter } from '../src';
+
+// TO DO: Replace with proper button import once published to npm
+import { Button } from '../../button/src';
```

Comment:
Should we write this TODO comment consistently... I've got many of these to switch over once published. Find/replace would be easier?

Reply:
I think it would be good to do it consistently so its easier to update once we start publishing packages

Reply:
Actually its not necessary. We can straight up install the dependency via bolt. Bolt creates a symlink to the local package. Pretty cool stuff :astonished: